shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -137,15 +171,18 @@
 
   layoutChanged = layoutChanged || [doomed_uids count] > 0;
   if (routeChanged) {
-    NSString* routeName = [lastAdded routeName];
-    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, routeName);
+    if (!ios_delegate_->IsFluterViewControllerPresentingModalViewController(view_)) {
+      NSString* routeName = [lastAdded routeName];
+      ios_delegate_->PostAccessibilityNotification(UIAccessibilityScreenChangedNotification,
+                                                   routeName);
+    }
   } else if (layoutChanged) {
     // TODO(goderbauer): figure out which node to focus next.
-    UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, nil);
+    ios_delegate_->PostAccessibilityNotification(UIAccessibilityLayoutChangedNotification, nil);
```

Comment:
Why run the notification through the delegate as well?

Reply:
For testing.  We can't test that someone has called UIAccessibilityPostNotification so the delegate allows us to do dependency injection.

Reply:
That makes sense. Thanks for the explanation :)